src/main/scala/mesosphere/marathon/api/MarathonExceptionMapper.scala

```diff
@@ -70,6 +70,8 @@ class MarathonExceptionMapper extends ExceptionMapper[Exception] {
       )
     case e: JsonParseException =>
       Json.obj("message" -> e.getOriginalMessage)
+    case e: JsonMappingException =>
+      Json.obj("message" -> "Please specify data in JSON format")
```

Comment:
This happens when you don't specify required data, and used to return: 
`No content to map due to end-of-input\n at [Source: [B@1457c640; line: 1, column: 1]`

I just found it not useful, but I'll put it back in so we don't lose any information.